### ISSUE TITLE
Attachment popup not showing since RC 1.3.5

### ISSUE THREAD

**Issue by jockie** [messagerie-melanie2/Roundcube-Plugin-Mobile#36]
Simply does not show. Shows some preventDefault-JS-Error in JS Console.

**Comment by ghost** [comment]
Please, test on roundcube 1.3.6.
Write you system info:
Like this:
https://github.com/6ec123321/chbox/blob/master/README
[TEST SERVER CONFIGURATION]
ubuntu 16.04 4.4.0-116-generic
Apache/2.4.18 (Ubuntu)
PHP 5.6.34-1+ubuntu16.04.1+deb.sury.org+1 (cli)
mysql  Ver 14.14 Distrib 5.7.21, for Linux (x86_64) using  EditLine wrapper

or

testing on this for fast:
https://github.com/6ec123321/docker-alpine-roundcubemail-1.3.6

i'm using this docker-image in 5 internet-servers

**Comment by futureweb** [comment]
1.3.8 - same problem when trying to open Attachment Popup ... 

TypeError: a.preventDefault is not a function

seems like an incompatibility between jquery mobile and jquery version ... 
Related to: https://github.com/jquery/jquery/issues/2917 I guess ...

mhm ?!?

**Comment by futureweb** [comment]
That's what I did to get File Upload working again (temporary fix):

replace minified Version with unminified:
- mv plugins/jquery_mobile/js/jquery.mobile-1.4.5.min.js plugins/jquery_mobile/js/jquery.mobile-1.4.5.min.js.orig
- cp plugins/jquery_mobile/js/jquery.mobile-1.4.5.js plugins/jquery_mobile/js/jquery.mobile-1.4.5.min.js.orig

Diff of Change:
```
# diff jquery.mobile-1.4.5.min.js jquery.mobile-1.4.5.js
3032c3032
<                               /*popstateEvent.originalEvent = new $.Event {
---
>                               popstateEvent.originalEvent = {
3035,3037c3035
<                               };*/
<                               popstateEvent.originalEvent = new $.Event( "popstate", { state: null } );
<                               state.id = ( this.squash( url, state ) || {} ).id;
---
>   
```

**Comment by tpayen** [comment]
Thanks, it seems to work well !
I check that everything works with this patch and I release a new version of jquery mobile plugin quickly

**Comment by tpayen** [comment]
Fixed in jquery_mobile 1.2 https://github.com/messagerie-melanie2/Roundcube-Plugin-JQuery-Mobile/releases/tag/v1.2
